spring-ai-docs/src/main/antora/modules/ROOT/pages/api/embeddings/vertexai-embeddings-palm2.adoc

```diff
@@ -1,6 +1,12 @@
-= VertexAI Embeddings
+= Google VertexAI PaLM2 Embeddings
```

Comment:
is this API being decommissioned this august?

Reply:
It is very confusing information. 

According to this message  https://ai.google.dev/palm_docs/deprecation it should have been deprecated in August. But then mind this note:
> Note that this deprecation notice does not pertain to PaLM API support in Vertex AI.

Same here: https://ai.google.dev/palm_docs/palm
> The PaLM API is deprecated for use with Google AI services and tools (BUT NOT FOR Vertex AI).

Not sure what the "Not for Vertex AI" means?